plugins/notify/build.sbt

```diff
@@ -2,7 +2,7 @@ organization := "com.github.dcshock"
 
 name := "forklift-notify"
```

Comment:
This plugin doesn't do anything and nobody uses it, right? Why bother keeping it around?